go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -474,6 +474,16 @@ func addDefaultConntrackRules(nodeName, gwBridge, gwIntf string, stopChan chan s
 		nFlows++
 	}
 
+	// table 0, packets coming from host should go out physical port
+	_, stderr, err = util.RunOVSOfctl("add-flow", gwBridge,
+		fmt.Sprintf("cookie=%s, priority=100, in_port=LOCAL, actions=output:%s",
+			defaultOpenFlowCookie, ofportPhys))
```

Comment:
I don't understand exactly what `LOCAL` means, but the comment, as written, seems wrong. What if the packet is coming from the host going to a local pod? Oh, or does that go to the management port without ever hitting this bridge?

Reply:
correct. any host to pod traffic gets SNAT'ed and sent through the managment port, no traffic from host -> ovn ever enter this bridge